ui/src/features/Dashboard/components/MainSection.tsx

```diff
@@ -33,7 +39,17 @@ const MainSection: React.FC = () => {
     }
   };
 
-  const projects = generateProjects(10);
+  // TODO: Using sample workflows at the moment
+  useEffect(() => {
+    if (!projects) return;
+    projects.forEach(p => {
+      p.workflow = generateWorkflows(1)[0];
+    });
+  }, [projects]);
+
+  const handleDeleteProject = async (id: string) => {
+    await deleteProject(id);
+  };
```

Comment:
Keep in mind we will eventually want a warning Dialog for actions like this. (once you read this you can resolve this comment yourself 👍🏻 )

Reply:
Added a todo for reference. Same as workspace